text/src/autogluon/text/text_prediction/presets.py

```diff
@@ -0,0 +1,239 @@
+        search_space['model.network.aggregate_categorical'] = False
+        search_space['preprocessing.categorical.convert_to_text'] = True
+        search_space['preprocessing.numerical.convert_to_text'] = True
+    else:
+        raise NotImplementedError
+    return new_cfg
+
+
+def apply_backbone(cfg, backbone_name='electra_base'):
+    new_cfg = copy.deepcopy(cfg)
+    search_space = new_cfg['models']['MultimodalTextModel']['search_space']
+    if backbone_name == 'electra_small':
+        search_space['model.backbone.name'] = 'google_electra_small'
+        search_space['optimization.per_device_batch_size'] = 16
+    elif backbone_name == 'electra_base':
+        search_space['model.backbone.name'] = 'google_electra_base'
+        search_space['optimization.per_device_batch_size'] = 8
+    elif backbone_name == 'electra_large':
+        search_space['model.backbone.name'] = 'google_electra_large'
+        search_space['optimization.per_device_batch_size'] = 4
+    elif backbone_name == 'roberta_base':
+        search_space['model.backbone.name'] = 'fairseq_roberta_base'
+        search_space['model.network.text_net.use_segment_id'] = False
+        search_space['optimization.per_device_batch_size'] = 8
+    elif backbone_name == 'multi_cased_bert_base':
+        search_space['model.backbone.name'] = 'google_multi_cased_bert_base'
+        search_space['optimization.per_device_batch_size'] = 8
+    else:
+        raise NotImplementedError
+    return new_cfg
+
+
+def gen_config_no_hpo(backbone_name, nbest=3, fusion_strategy='fuse_late'):
+    cfg = base()
+    cfg = apply_backbone(cfg, backbone_name)
+    cfg = apply_average_nbest(cfg, nbest=nbest)
+    cfg = apply_fusion_strategy(cfg, strategy=fusion_strategy)
+    return cfg
+
+
+for backbone_name in ['electra_small', 'electra_base',
+                      'electra_large', 'roberta_base',
+                      'multi_cased_bert_base']:
+    for fusion_strategy in ['fuse_late']:
+        ag_text_presets.register(f'{backbone_name}_{fusion_strategy}',
+                                 functools.partial(gen_config_no_hpo,
+                                                   backbone_name=backbone_name,
+                                                   nbest=3,
+                                                   fusion_strategy=fusion_strategy))
+
+
+for backbone_name in ['electra_base']:
+    for fusion_strategy in ['fuse_early', 'all_text']:
+        ag_text_presets.register(f'{backbone_name}_{fusion_strategy}',
+                                 functools.partial(gen_config_no_hpo,
+                                                   backbone_name=backbone_name,
+                                                   nbest=3,
+                                                   fusion_strategy=fusion_strategy))
+
+
+@ag_text_presets.register()
+def default() -> dict:
+    return ag_text_presets.create('electra_base_fuse_late')
+
+
+@ag_text_presets.register()
+def fast_train() -> dict:
```

Comment:
Can we standardize preset names with Tabular?

If standardized, this preset should be called: `medium_quality_faster_train`